resources/lib/KodiHelper.py

```diff
@@ -995,16 +995,19 @@ def _generate_art_info(self, entry):
             'landscape': '',
             'thumb': '',
             'fanart': '',
-            'poster': ''
+            'poster': '',
+            'clearlogo': ''
         })
         if 'boxarts' in dict(entry).keys() and not isinstance(entry.get('boxarts'), dict):
             big = entry.get('boxarts', '')
             small = big
+            poster = big
         if 'boxarts' in dict(entry).keys() and isinstance(entry.get('boxarts'), dict):
             big = entry.get('boxarts', {}).get('big')
             small = entry.get('boxarts', {}).get('small')
+            poster = entry.get('boxarts', {}).get('poster')
             art.update({
-                'poster': big or small,
```

Comment:
should be: poster or big or small

Reply:
I can't guarantee it, but all shows I checked have had a poster available.
If there's no poster in the expected dimensions available, it should rather be left empty anyway, as wrong aspect ratios break some skins.